packages/react-devtools/app.js

```diff
@@ -32,6 +32,12 @@ app.on('ready', function() {
     },
   });
 
+  // https://stackoverflow.com/questions/32402327/
+  mainWindow.webContents.on('new-window', function(event, url) {
+    event.preventDefault();
```

Comment:
Looks like [`new-window`](https://github.com/electron/electron/blob/master/docs/api/web-contents.md#event-new-window-deprecated) is deprecated in favor of [`webContents.setWindowOpenHandler`](https://github.com/electron/electron/blob/master/docs/api/web-contents.md#contentssetwindowopenhandlerhandler).

Reply:
Nice tip! Thanks :)

Reply:
React DevTools is using Electron 11.1.0. The `webContents.setWindowOpenHandler` API was added in 12.0 beta (https://github.com/electron/electron/pull/24517). Thanks though!